### PR TITLE
Remove special characters in Faker::Internet

### PR DESCRIPTION
**Pull request by vbrazo**
This PR fixes https://github.com/igas/faker/issues/199.

I've added:

- [ ] USAGE.md docs if applicable
- [x] CHANGELOG.md

### CODE REVIEW

**Comment by igas** [comment]
@vbrazo hey, sorry for misguiding you in the issue. While the original issue was not 100% accurate, I think it makes sense to make it more generic. One thing is I would rather go from a small number of exceptions like only disallow `'` & `"` for now. I think having `-` or `_` quite common in emails.

Also, can you please add a test like https://github.com/igas/faker/blob/c36d9ee28b244c4b9a9b235f51ca642bff7d37e0/test/faker/app_test.exs for every public function that returns email address to not contain quotes?

**Comment by vbrazo** [comment]
Actually the [CI issue](https://travis-ci.org/igas/faker/jobs/433497516#L627) is related to an Error while running `Elixir.Credo.Check.Consistency.SpaceInParentheses`.

**Comment by igas** [comment]
Interesting, it's the same in master and passes...

File: test/faker/internet_test.exs [hunk 1]
```diff
@@ -37,4 +41,44 @@ defmodule Faker.InternetTest do
 
     assert Regex.match?(~r/\A((hello|world)-)(hello|world)\z/, slug)
   end
+
+  test "user_name/0" do
+    Stream.repeatedly(&user_name/0)
+    |> Enum.take(@iterations)
+    |> Enum.each(fn generated_value ->
+      assert Regex.match?(~r/^((?!").)|((?!').)*$/, generated_value)
```
Comment (igas) [inline test/faker/internet_test.exs]: how about `refute`? So we are actually checking that we don't have quotes? like:

```
refute String.contains?(generated_value, ~s('"))
```

**Pull request merged**